mica/starcheck/tests/test_catalog_fetches.py

```diff
@@ -154,14 +155,14 @@ def test_obsid_catalog_fetch():
         {"obsid": 17210, "mp_dir": "/2016/JAN2516/oflsa/", "n_cat_entries": 11},
         {"obsid": 62668},
     ]
-    # 62668 should be an undercover with no catalog
+    # 45312 is a gyro hold with no star catalog
```

Comment:
These tests are awkward but if this code should be called on 45312 I think 45312 needs to be added to tests above and 62668 should be removed.

Reply:
Fixed.